ui/src/pages/version/components/forms/components/transformer/components/table_outputs/UpiPreprocessOutputCard.js

```diff
@@ -0,0 +1,89 @@
+import {
+  EuiFlexGroup,
+  EuiPanel,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiFieldText,
+  EuiToolTip,
+  EuiText,
+  EuiSpacer,
+  EuiIcon
+
+} from "@elastic/eui";
+import {
+  get, 
+  useOnChangeHandler
+} from "@gojek/mlp-ui";
+import {ColumnsComboBox} from "../table_operations/ColumnsComboBox"
+import { DraggableHeader } from "../../../DraggableHeader";
+
+
+export const UpiPreprocessOutputCard = ({ output, onDelete, onChangeHandler, errors = {}, ...props }) => {
+  const { onChange } = useOnChangeHandler(onChangeHandler);
+  return (
+    <EuiPanel>
+      <DraggableHeader
+        onDelete={onDelete}
+        dragHandleProps={props.dragHandleProps}
+      />
+      <EuiSpacer size="s" />
+      <EuiFlexGroup direction="column" gutterSize="m">
+        <EuiFlexItem>
+          <EuiToolTip content="UPI Preprocess Output will create output in UPI PredictionValueRequest type">
+            <span>
+            <EuiText size="s">
+              <h4>UPI Preprocess Output</h4>
+            </EuiText>
+              
+            </span>
+          </EuiToolTip>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFormRow
+            label={
+              <EuiToolTip content="Prediction Table Name contains instances to be predicted. This should contain all preprocessed feature that model use to perform prediction">
```

Comment:
Should be 
`Table name that will be used to populate prediction_table field in the request to model`